### PR TITLE
test: CacheLabel

### PR DESCRIPTION
**Pull request by yardz**
### SUMMARY
- Extended the label props to HTMLAttributes (avoid errors for invalid props when they are actually valid)
- Spliting the content of tolltip into another component: `TolltipContent`.
- Converting component to typescript.
- Creating tests for these components

### TEST PLAN
No change in behavior. All tests must pass.

### CODE REVIEW

File: superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import moment from 'moment';
+import { TooltipContent } from './TooltipContent';
+
+test('Rendering TooltipContent correctly - no timestep', () => {
```
Comment (geido) [inline superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx:25]: Small typo timestep to be timestamp

File: superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  render(<TooltipContent />);
+  expect(screen.getByTestId('tooltip-content')?.textContent).toBe(
+    'Loaded from cache. Click to force-refresh',
+  );
+});
+
+test('Rendering TooltipContent correctly - with timestep', () => {
```
Comment (geido) [inline superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx:32]: Same typo. Found also in the other tests

**Review by geido — APPROVED** [review]
LGTM!

File: superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx (hunk 1 above)
Comment (yardz) [reply superset-frontend/src/components/CachedLabel/CachedLabel.test.tsx]: Copy and paste error rsrsrs
Thanks

**Comment by yardz** [comment]
@ktmud or @rusackas

**Review by pkdotson — APPROVED** [review]

**Review by rusackas — APPROVED** [review]

**Pull request merged**